### DISCUSSION TITLE
Workflow identification

### DISCUSSION THREAD

**Discussion by ZonSoft** [Comfy-Org/ComfyUI-Manager#51] [Ideas]
May I suggest some workflow identification in manager - at least if would be very convenient to see the name of current workflow.
If it would be possible to pre-fill SAVE dialog with this name instead of "workflow.json", that would be also very useful, but that's extra...

**Comment by ltdrdata** [comment]
I think your suggestion should be posted to here: 

https://github.com/comfyanonymous/ComfyUI

edit: oops... wrong link.

**Reply by ZonSoft** [reply]
Sorry, wrong place again... I did believe Manager is the best place to place such info.

**Comment by ZonSoft** [comment]
__init__.py line 266 - "FECTH " instead of "FETCH"
